client/lib/feed-stream-store/actions.js

```diff
@@ -11,6 +11,8 @@ import { action as ActionType } from './constants';
 import FeedPostStoreActions from 'lib/feed-post-store/actions';
 import feedPostListCache from './feed-stream-cache';
 import wpcom from 'lib/wp';
+import { reduxDispatch } from 'lib/redux-bridge';
```

Comment:
i initially called it dispatch, but it seems to end up in the same file as the flux dispatcher and this just seemed clearer. I also sort of want to make this ugly to use on purpose.